sumologic/resource_sumologic_cse_log_mapping.go

```diff
@@ -0,0 +1,462 @@
+				Required: true,
+			},
+			"parent_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"product_guid": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"record_type": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"relates_entities": {
+				Type:     schema.TypeBool,
+				Optional: true,
+			},
+			"skipped_values": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"fields": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+
+						"name": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"value": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"value_type": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"skipped_values": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"default_value": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"format": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"case_insensitive": {
+							Type:     schema.TypeBool,
+							Optional: true,
+						},
+						"alternate_values": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"time_zone": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"split_delimiter": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"split_index": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"field_join": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"join_delimiter": {
+							Type:     schema.TypeString,
+							Optional: true,
+						},
+						"format_parameters": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+						"lookup": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+
+									"key": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"value": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"structured_inputs": {
+				Type:     schema.TypeList,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: getLogMappingStructuredInputFieldSchema(),
+				},
+			},
+			"unstructured_fields": {
```

Comment:
`structuredFields` is missing here. Is that field not supported?

Reply:
it is a deprecated field, structured_inputs usage instead